i18n/en/pop_desktop_widget.ftl

```diff
@@ -19,7 +19,7 @@ display-primary = Primary Display
 
 dock-always-hide = Always hide
 dock-always-hide-description = Dock always hides unless actively being revealed by the mouse
-dock-always-visible = Always visible
+dock-always-visible = Always show
```

Comment:
The other two options describe an action (hide), but this option describes a passive state. It is mentally simpler if all three options describe an action.
Languages other than English can also construct sentences differently, depending on whether it is a state or action, and that can make the difference even more mentally challenging.